README.md

```diff
@@ -21,6 +21,18 @@ LIGHTUP_TLA=dum LIGHTUP_TOKEN=e3976956-9ef0-44c5-a978-2ee9149e1234 LIGHTUP_INSTA
 
 ## Troubleshooting and Overrides
 
+### Proxy Support
+
+If tunnel needs to be run through an HTTP/HTTPS proxy
+
+1. Update `LIGHTUP_CONNECT_SERVER_PORT` in fixed_config.sh to indicate either 80 or 443 based on the type of proxy.
+2. Run the `proxy.sh` script with the appropriate environment variables (Required: PROXY_HOST, PROXY_PORT. Optional: PROXY_USER, PROXY_PASSWORD)
+
+Example:
+```
+LIGHTUP_TLA=dum PROXY_HOST=proxy.dum.com PROXY_PORT=8192 ./proxy.sh
```

Comment:
I'd suggest removing this separate script and just sticking with bootstrap.sh and connect.sh.

Reply:
Will do this in another PR.